modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -86,3 +87,16 @@ module.exports = ({
 
   return merge(config, ...tasks);
 };
+
+function getVueAlias() {
+  if (!process.env.APOS_DEV) {
+    return '@vue/runtime-dom';
+  }
```

Comment:
```suggestion
function getVueAlias(config) {
  if (config.mode !== 'development') {
    return '@vue/runtime-dom';
  }
```

this uses `NODE_ENV`, not `APOS_DEV`